lib/scraper_test.rb

```diff
@@ -30,17 +31,41 @@ def install_tasks
           it 'should contain the expected data' do
             # TODO: Make this configurable
             Dir['test/data/*.yml'].each do |file|
-              yaml_data = YAML.load_file(file).to_h
-              url = yaml_data[:url]
-              class_to_test = Object.const_get(yaml_data[:class])
-              VCR.use_cassette(File.basename(url)) do
-                response = class_to_test.new(response: Scraped::Request.new(url: url).response)
-                response.to_h.must_equal yaml_data[:to_h]
+              VCR.use_cassette(File.basename(file)) do
+                test = Instructions.new(file)
+                response = test.class_to_test.new(response: Scraped::Request.new(url: test.url).response)
+                response.to_h.must_equal test.expected
```

Comment:
will be good to get rid of these too, but that can certainly happen in a separate PR.